index.js

```diff
@@ -143,7 +151,7 @@ async function run() {
 
       analyzer.analyze(pattern);
 
-      await exec.exec('git', ['switch', '-'], { cwd: mainRepoPath, stdio: 'inherit' });
+      await exec.exec('git', ['switch', pr.head.ref], { cwd: mainRepoPath, stdio: 'inherit' });
```

Comment:
so is that a fix to our current issue when the tests are not shown on PR?

Reply:
Yes